model/task/db.go

```diff
@@ -631,6 +631,49 @@ func GetRecentTasks(period time.Duration) ([]Task, error) {
 	return tasks, nil
 }
 
+type TaskByProjectList struct {
```

Comment:
I meant to remove the changes to this file. I'm using some of Yves changes instead.